pkgs/development/ocaml-modules/zbg/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildDunePackage,
+  ocamlPackages,
+}:
+buildDunePackage {
+  pname = "zbg";
+  version = "0.2.0";
+
+  minimalOCamlVersion = "3.7";
+
+  buildInputs = with ocamlPackages; [
+    ansiterminal
+    core
+    core_unix
+    ppx_assert
+    ppx_deriving
+    ppx_inline_test
+    ppx_jane
+  ];
+
+  src = fetchFromGitHub {
+    owner = "chshersh";
+    repo = "zbg";
+    rev = "9fa381bd0301027e5ac25ef157b6c0075c00f6f2";
```

Comment:
can you reference the upstream tag instead?

Reply:
Upstream tag is outdated, which is why I didn't use it. The project seems mature enough (no commit in quite some time) so using a revision is fine here imo. If the package gets updated upstream by any chance, I'll change here accordingly.

Reply:
Then we probably would signal that in the version

Reply:
Marking the package as "unstable" is possible, but it feels a little excessive. The package hasn't been updated in 2 years, and the commits after the latest tag are more into the patch territory.

Should I still go mark it as 0-unstable-date?